src/gui/windows/debug.zig

```diff
@@ -61,6 +67,21 @@ pub fn render() void {
 		y += 8;
 		draw.print("Queue size: {}", .{main.threadPool.queueSize()}, 0, y, 8, .left);
 		y += 8;
+		const perf = main.threadPool.performance.read();
+		const values = comptime std.enums.values(TaskType);
+		var totalUtime: i64 = 0;
+		for(values) |task|
+			totalUtime += perf.utime[@intFromEnum(task)];
+		draw.print("Total task time: {} µs", .{totalUtime}, 0, y, 8, .left);
```

Comment:
To me printing the total time seems kind of useless. It's just kind of a random number.